bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/robot/RRMapFileParser.java

```diff
@@ -0,0 +1,415 @@
+                        walls.add(new Float[] { x0, y0, x1, y1 });
+                    }
+                    break;
+                case NO_GO_AREAS:
+                case MFBZS_AREA:
+                    int areaPairs = getUInt16(header, 0x08);
+                    ArrayList<Float[]> area = new ArrayList<Float[]>();
+                    for (int areaPair = 0; areaPair < areaPairs; areaPair++) {
+                        Float x0 = offset - (getUInt16(raw, blockDataStart + areaPair * 16)) / MM;
+                        Float y0 = getUInt16(raw, blockDataStart + areaPair * 16 + 2) / MM - top;
+                        Float x1 = offset - (getUInt16(raw, blockDataStart + areaPair * 16 + 4)) / MM;
+                        Float y1 = getUInt16(raw, blockDataStart + areaPair * 16 + 6) / MM - top;
+                        Float x2 = offset - (getUInt16(raw, blockDataStart + areaPair * 16 + 8)) / MM;
+                        Float y2 = getUInt16(raw, blockDataStart + areaPair * 16 + 10) / MM - top;
+                        Float x3 = offset - (getUInt16(raw, blockDataStart + areaPair * 16 + 12)) / MM;
+                        Float y3 = getUInt16(raw, blockDataStart + areaPair * 16 + 14) / MM - top;
+                        area.add(new Float[] { x0, y0, x1, y1, x2, y2, x3, y3 });
+                    }
+                    areas.put(Integer.valueOf(blocktype & 0xFF), area);
+                    break;
+                case OBSTACLES:
+                    int obstaclePairs = getUInt16(header, 0x08);
+                    for (int obstaclePair = 0; obstaclePair < obstaclePairs; obstaclePair++) {
+                        int x0 = getUInt16(data, obstaclePair * 5 + 0);
+                        int y0 = getUInt16(data, obstaclePair * 5 + 2);
+                        int u = data[obstaclePair * 5 + 0] & 0xFF;
+                        obstacles.add(new int[] { x0, y0, u });
+                    }
+                    break;
+                case BLOCKS:
+                    int blocksPairs = getUInt16(header, 0x08);
+                    blocks = getBytes(data, 0, blocksPairs);
+                    break;
+                default:
+                    logger.info("Unknown blocktype (pls report to author)");
+                    printBlockDetails = true;
+            }
+            if (logger.isTraceEnabled() || printBlockDetails) {
+                logger.info("Blocktype: {}", Integer.toString(blocktype));
+                logger.info("Header len: {}   data len: {} ", Integer.toString(blockHeaderLength),
+                        Integer.toString(blockDataLength));
+                logger.info("H: {}", Utils.getSpacedHex(header));
+                if (blockDataLength > 0) {
+                    logger.info("D: {}", (blockDataLength < 60 ? Utils.getSpacedHex(data)
+                            : Utils.getSpacedHex(getBytes(data, 0, 60))));
+                }
+                printBlockDetails = false;
+            }
```

Comment:
it is because of the ` || printBlockDetails  `
If trace is enabled, I want to see the full lock. If no trace, I only want to see the unknown blocktypes. (which currently should be none, but the format is sometimes extended.
I'll change it to debug, to avoid excessive logging to INFO